fvtr/tbb/tbb.exp

```diff
@@ -105,8 +105,8 @@ set tmp_file [exec mktemp]
 # Run memtest for all TBB versions.
 memtest
 
-# parallel_test is supported by TBB >= 4.2 only.
-if { ${major_ver} >= 4 && ${minor_ver} >= 2 } {
+# parallel_test is supported by TBB >= 4.3 only.
+if { ${major_ver} >= 4 && ${minor_ver} >= 3 } {
```

Comment:
Prior to 4.2u4 we would need the tbb/internal/_flow_graph_or_impl.h header.

Reply:
It would be nice if you moved this comment (on github.com) to the code itself.
Other than that, the patch is OK.